openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
I just gave this a quick look but this appears to be incorrect or maybe I missed something.  It looks like you never instantiate the subclass 'TimeMod' anywhere. Looks to me that everything in `runTimes` are all `Time` and not the subclass `TimeMod` (https://github.com/simphony/simphony-openfoam/pull/58/files#diff-dcbcfcd124d66fd0f6a13adb92495896R67)